pkgs/development/libraries/glib/default.nix

```diff
@@ -56,6 +56,12 @@ stdenv.mkDerivation rec {
   };
 
   patches = optional stdenv.isDarwin ./darwin-compilation.patch
+    ++ optional stdenv.isDarwin (fetchpatch {
```

Comment:
Can we make the above patch use `optionals` so this patch can be in a list instead of another `optional`?